source/integrations/index.html

```diff
@@ -23,43 +23,87 @@
 {%- assign components = site.integrations | sort: 'title' -%}
 {%- assign components_by_version = site.integrations | group_components_by_release -%}
 {%- assign categories = components | map: 'ha_category' | join: ',' | join: ',' | split: ',' | uniq | sort -%}
+{%- assign iot_classes = components | map: 'ha_iot_class' | join: ',' | join: ',' | split: ',' | uniq | sort -%}
+{%- assign quality_scales = components | map: 'ha_quality_scale' | join: ',' | join: ',' | split: ',' | uniq | sort  -%}
 
 <p class='note'>
   Support for these integrations is provided by the Home Assistant community.
 </p>
 <div class="grid">
-  <div class="grid__item one-sixth lap-one-whole palm-one-whole">
-
-    <div class="filter-button-group">
-      <a href='#all' class="btn">All ({{tot}})</a>
-      <a href='#featured' class="btn featured">Featured</a>
-      <div class="version_select">Added in: <select name="versions">
-          <option value="#"></option>
-          {%- for group in components_by_version -%}
-          <optgroup label="{{ group.label }} ({{group.new_components_count}})">
-            {%- for version in group.versions -%}
-            <option value="#version/{{ version.label }}">{{ version.label }} ({{ version.new_components_count }})
-            </option>
-            {%- endfor -%}
-          </optgroup>
+  <div class="grid__item six-sixths lap-one-whole palm-one-whole">
+    <div class="component-search">
+      <form onsubmit="event.preventDefault(); return false">
+        <input type="text" name="search" id="search" data-id="search" class="search" placeholder="Search integrations..." autofocus />
+      </form>
+    </div>
+  </div>
+  <div class="grid__item three-sixths lap-one-whole palm-one-whole">
```

Comment:
I think `one-quarter` looks better:
![image](https://user-images.githubusercontent.com/15093472/203175665-4200c2e8-1dfc-4e8f-aa6b-357a59d99558.png)
`three-sixths`:
![image](https://user-images.githubusercontent.com/15093472/203175683-d0bae818-199d-4841-8657-d734a64e3c15.png)

Reply:
updated to use `one-quarter` class for select dropdown width